### PR TITLE
Interactions

### PR DESCRIPTION
**Pull request by Apatakiti**
I think I'm done with all the procedure listed below.

- Add a new JavaScript file and import it as a module:
- it will contain methods related to the status updates (completed: true / false).
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button (use filter() method).
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        this.delete(eachList[lis.index]);
+      });
+
+      eachList[lis.index].addEventListener('mouseleave', (eve) => {
+        eachbtn[lis.index].innerHTML = '<i class="fa-solid fa-trash"></i>';
+        this.update(eve.target.id, eve.target.innerText);
+      });
+
+      // Listen for checking elements
+      eachcheck[lis.index].addEventListener('change', (e) => {
```
Comment (tufoinnkuo10) [inline src/index.js:79]: - [x] Great job adding all the interactions. Everything works fine. Kindly ensure that when a checkbox is checked, the completed status turns to true. Some change and some don't. Kindly look into it. 
Screenshot (295)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Ademola-coding 
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ clear all button works as expected

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/index.js (hunk 1 above)
Comment (Apatakiti) [reply src/index.js:79]: Screenshot (421)
DONE

Comment (Apatakiti) [reply src/index.js:79]: DONE

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Ademola-coding,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**